### ISSUE TITLE
Pattern with plus not works as expected

### ISSUE THREAD

**Issue by NAlexandrov** [micromatch/nanomatch#13]
```javascript
const nanomatch = require('nanomatch');

const soft = ['Microsoft Visual C++ 2013 x86 Additional Runtime - 12.0.21005'];
const rule = '*Microsoft Visual C++*';

const match = nanomatch.match(soft, rule);

console.log(match);
```

## What is happening (but shouldn't):
`match = [];`

## What should be happening instead?
`match = ['Microsoft Visual C++ 2013 x86 Additional Runtime - 12.0.21005'];`

Why?

P.S. All will be as expect if you change `require('nanomatch')` to `require('minimatch')`

**Comment by jonschlinkert** [comment]
> P.S. All will be as expect if you change require('nanomatch') to require('minimatch')

Oh cool, you found something that minimatch doesn't fail on ;)

**Comment by jonschlinkert** [comment]
fixed, published and [released as 1.2.11](https://github.com/micromatch/nanomatch/releases/tag/1.2.11). (In case anyone is wondering why a patch version is missing, I pubilshed 1.2.10 initially and noticed a bug right after I hit enter... So I unpublished 1.2.10, fixed the bug and republished as 1.2.11)

Thanks for reporting the issue @NAlexandrov!
